.github/workflows/export-notion-blocks-and-commit.yml

```diff
@@ -15,15 +15,16 @@ jobs:
     timeout-minutes: 35
     steps:
       - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
+        with:
+          version: '7'
```

Comment:
use version 7 taken from example https://github.com/karlhorky/archive-webpage-browser-extension/blob/main/.github/workflows/lint-check-types-and-build.yml